cpp/data/system_updates/123.vufind.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE tuefind_publications ADD doi_link VARCHAR(255) NULL AFTER external_document_guid;
+ALTER TABLE tuefind_publications ADD doi_notification DATE NULL AFTER doi_link;
```

Comment:
Maybe use DATETIME instead?

Reply:
(with DEFAULT NOW()? then maybe you dont even have to set an explicit value in the query)